src/test/lib/pgSQL-tests.js

```diff
@@ -0,0 +1,425 @@
+/**************************************************************
+ *
+ * UNIT TESTS postgresQL
+ *
+ **************************************************************/
+
+// external modules
+const assert = require('assert');
+const config = require('../../config/config');
+const async = require('async');
+pg=require('../../lib/postgresQL')(config.pg);
```

Comment:
Add a `const` before `pg` and spacing around the equation
```suggestion
const pg = require('../../lib/postgresQL')(config.pg);
```